### PR TITLE
Homework: nwd-nnw

### PR DESCRIPTION
**Pull request by mariuszlisowski**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @mariuszlisowski

**Comment by ziobron** [comment]
⏱ 3 XP extra for @mariuszlisowski for on-time delivery

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,20 @@
 #pragma once
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    int remainder;
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:4]: Better always initialize values created on stack with default value.

**Review by nauka-programowania-MA — APPROVED** [review]

File: homework/nwd-nnw/nwdNww.hpp (hunk 1 above)
Comment (mariuszlisowski) [reply homework/nwd-nnw/nwdNww.hpp:4]: Initialized with curly braces

**Pull request closed without merging**